svc-bip-api/src/main/resources/application.yaml

```
@@ -40,6 +40,8 @@ createClaimContentionsQueue: createClaimContentionsQueue
 updateClaimContentionsQueue: updateClaimContentionsQueue
 putTempStationOfJurisdictionQueue: putTempStationOfJurisdictionQueue
 exchangeName: bipApiExchange
+deadLetterQueueName: bipDeadLetterQueue
```

```suggestion
deadLetterQueue: bipDeadLetterQueue
```
for consistency with the variables for queue names in the lines above